src/core/agent/services/identifierService.ts

```diff
@@ -226,6 +227,16 @@ class IdentifierService extends AgentService {
     );
   }
 
+  async getSigner(identifier: string): Promise<Signer> {
+    const metadata = await this.getMetadataById(identifier);
+    this.validIdentifierMetadata(metadata);
```

Comment:
You need to check that it's a KERI identifier and that `metadata.signifyName` is set